README.md

```diff
@@ -79,74 +78,83 @@ After that you have the following instance methods available to you.
 - ### `national_number(formatted: true)`
 
   Returns the national formatted number including special characters such as parenthesis and dashes. You can omit the special characters by passing `formatted: false`
-  
+
   `phone_object.national_number ==> "(317) 508-2237"`
 
 - ### `international_number(formatted: true)`
-  
+
   Returns the international formatted number including special characters such as parenthesis and dashes. You can omit the special characters by passing `formatted: false`
 
   `phone_object.international_number ==> "+1 317-508-2237"`
-  
+
 - ### `e164_number(formatted: true)`
 
   Returns the international formatted number including special characters such as parenthesis and dashes. You can omit the special characters by passing `formatted: false`
-  
+
   `phone_object.e164_number ==> "+13175082237"`
-  
+
 - ### `country`
-  
+
   Returns an object containing data related a the number's country.
-  
+
   ```
-  phone_object.country ===> 
+  phone_object.country ===>
    #<TelephoneNumber::Country:0x007fb976267410
    @country_code="1",
    @country_id="US",
    ...
   ```
-  
+
+- ### `location(:en)`
+
+  Returns the geolocation of the number.
```

Comment:
Should we just say "location" as opposed to "geolocation"?

Reply:
Yea. I went back and forth on it trying to decide if it was better to tie it to the method name or reference the file names.